calcom/cal.com
# fix: Fallback to the default language (en) on Admin page in User edit form

## What does this PR do?
When the user's locale value wasn't specified the User Form on Admin page would fallback to the first locale in the alphabetical order which would be Arabic. Now it's falling back to English which is a correct default value for the app.

- Fixes CAL-3011 

### BEFORE
<img width="1200" alt="Screenshot 2024-08-30 at 12 34 40" src="https://github.com/user-attachments/assets/b234dd54-e08e-4a53-ae73-8cff6026d6c8">

### AFTER
<img width="1200" alt="Screenshot 2024-08-30 at 12 29 06" src="https://github.com/user-attachments/assets/94475166-1d76-41ec-bdc6-36bc3ea1d8c3">

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
You need to be an admin of an organization and navigate to to `/settings/admin/users`, then if you edit one of the users that doesn't have the locale property set you should see English in the dropdown instead of Arabic.
